### PR TITLE
apple add mach_vm_map, mmap does not provide custom alignment so

### PR DESCRIPTION
**Pull request by devnexen**
mach_vm_map is only way.

### CODE REVIEW

**Comment by JohnTitor** [comment]
We've deprecated mach-related items since #1364 and a change like this should go to the `mach` crate, I think.

**Comment by devnexen** [comment]
I thought that but the crate had not been updated since 2019

**Review by JohnTitor — APPROVED** [review]
Uhm, that's unfortunate, we definitely should consider not suggesting that crate on the deprecation note and open an issue to discuss further.
Anyway, this PR should be fine then, thanks for clarifying!

File: libc-test/semver/apple.txt [hunk 1]
```diff
@@ -1700,6 +1700,7 @@ mach_port_t
 mach_thread_self
 mach_timebase_info
 mach_timebase_info_data_t
+mach_vm_map
```
Comment (JohnTitor) [inline libc-test/semver/apple.txt:1703]: Ah, the list seems incomplete.

**Pull request merged**